### ISSUE TITLE
مشکل پر شدن رم در نسخه 5.2

### ISSUE THREAD

**Issue by The-Binary-0-1** [radkesvat/ReverseTlsTunnel#155]
ارادت. آپدیت کردم به نسخه جدید ولی رو سرور ایران فقط یه تست با پهنای باید بالا مثلا 50 مگابیت کافیه که یهو رم از 318 مگ تا 800 مگ پر بشه!!!
سرور خارج فعلا مشکلی ندیدم. سپاس

**Comment by radkesvat** [comment]
سلام یه مشکل دیگه ای هست چون من بارها اسپید تست گرفتم تغییری در رم دیده نشده در سرور ایران

**Comment by The-Binary-0-1** [comment]
خب ظاهرا برنامه یه حجمی حدود 500 مگ از رم رو اشغال میکنه و بعد از اون چند دفعه تست میگیرم بالاتر نمیره. بعد از مدتی یه مقدار کمی از رم رو آزاد میکنه ولی نه مقدار قابل توجهی.
![Screenshot 2023-10-03 224433](https://github.com/radkesvat/ReverseTlsTunnel/assets/117812593/162d8a60-bd2c-4e1b-90ea-21d303b3f66d)

**Comment by radkesvat** [comment]
درسته ؛‌این دستورو میتنید بزنید و نتیجشو برام بفرستید ؟ مقدار دقیق رم رو با
> sudo pmap $(pidof RTT)

**Comment by The-Binary-0-1** [comment]
![Screenshot_20231003_233448_Termius](https://github.com/radkesvat/ReverseTlsTunnel/assets/117812593/e915a255-ac13-4805-9225-7131eb145408)
خدمت شما

**Comment by radkesvat** [comment]
عجیبه برای شما نمیدونم چرا اینقدر زیاد رم گرفته ؛ قابلیت یا سوییچ خاصی رو فعال کردین ...؟

**Comment by The-Binary-0-1** [comment]
ExecStart=/root/RTT --iran --lport:443 --password:12345 --keep-ufw --sni:www.soft98.ir --terminate:6 --noise:9 --mux-width:1 --log:0

یوزر ۳ تا. مصرف خاصی روش نیست.

**Comment by radkesvat** [comment]
خوب همین
--noise:9 

واقعا نیازه ؟ پهنای باند زیاد میخواد این و اینکه توی همین نسخه که منتشر شده استیبل ترش کردم ولی فکر نمیکردم کسی استفاده کنه این قابلیتو به اون صورت ؛ درسته دلیل رم پر کردین همینه

**Comment by The-Binary-0-1** [comment]
بله. واسه آپلود فیک اضافه کردم و متاسفانه دیتاسنتر گیر میده که یک به ده باشه وگرنه پهنای باند رو محدود می‌کنه و اخطار و ......
قابلیت خیلی خوبیه. دستت درد نکنه و البته خیلی ها هم استفاده میکنن به نظرم. ولی توی نسخه قبلی اینطوری نبود. حالا نمی‌دونم دقیقا چیکار کردین توی این نسخه....

**Comment by radkesvat** [comment]
اره درسته ؛ امشب یه اپدیت میدم استیبل تر میشه خیلی زود ؛ توی نسخه 5.2 یه حرکت زدم فکر کردم بهترش میکنه ولی تاثیر اش انگار خوب نبود و باعث پر شدن رم میشه

**Comment by radkesvat** [comment]
نسخه 5.3 منتشر شد لطفا اپدیت کنید و بگین که آیا تغییری کرده یا خیر

**Comment by The-Binary-0-1** [comment]
> نسخه 5.3 منتشر شد لطفا اپدیت کنید و بگین که آیا تغییری کرده یا خیر

بله. ظاهرا مشکل حل شده. بازم تست میگیرم موردی بود اطلاع میدم. ممنون.
